4. Objects and Functions.js

```diff
@@ -1,68 +1,63 @@
 /*Создать объект car, добавить к нему свойство color со значением 'черный'
 Изменить свойство color объекта car на 'зеленый'
 В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя*/
-
 let car = {
-    color: "black",
+  color: "black",
 };
 car.color = "green";
 car.power = prompt("enter the power");
 
-
-/* В терминале оплаты сохранено ваше имя, напишите функцию для определения имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)
-*/
+/* В терминале оплаты сохранено ваше имя, напишите функцию для определения 
+имени в терминале(если вы ввели ваше имя, то привет + имя, если нет, то нет такого имени)*/
 
 function showMessage() {
-    let userName = prompt('Enter your name', "Name");
-    if (userName == 'Aleksandr'){
-      var message = 'Hello, ' + userName +"!";
-    }else{
-      var message = 'sorry, no such name';
-    };
-    alert(message);
-  };
-  
-showMessage();
+  let userName = prompt("Enter your name", "Name");
+  if (userName == "Aleksandr") {
+    var message = "Hello, " + userName + "!";
+  } else {
+    var message = "sorry, no such name";
+  }
+  alert(message);
```

Comment:
No prompts or alerts in your code. Please use direct pass value in your code and logging to console.